generator/src/main/kotlin/ch/bailu/gtk/builder/ModelBuilder.kt

```diff
@@ -43,7 +44,7 @@ class ModelBuilder : BuilderInterface {
         var out: Writer? = null
         try {
             out = getCWriter(model, namespace)
-            model.write(CWriter(TextWriter(out)))
+            model.write(CWriter(TextWriter(out), this.errorStubs))
```

Comment:
Here you could just call 'genCCode' from 'namespace' and run 'model.write(CWriter(TextWriter(out))) conditionally. 

Its much simpler to just not write c-code here. Java code will still compile and run. Once a native function with no JNI backend gets called it will throw 'UnsatisfiedLinkError'.